src/core/sprites/canvas/CanvasTinter.js

```diff
@@ -1,250 +1,268 @@
 var utils = require('../../utils'),
-canUseNewCanvasBlendModes = require('../../renderers/canvas/utils/canUseNewCanvasBlendModes');
-/**
- * Utility methods for Sprite/Texture tinting.
- * @namespace PIXI.CanvasTinter
- */
-var CanvasTinter = {};
-module.exports = CanvasTinter;
+    canUseNewCanvasBlendModes = require('../../renderers/canvas/utils/canUseNewCanvasBlendModes');
 
 /**
- * Basically this method just needs a sprite and a color and tints the sprite with the given color.
+ * Utility methods for Sprite/Texture tinting.
  *
- * @memberof! PIXI.CanvasTinter#
- * @param sprite {PIXI.Sprite} the sprite to tint
- * @param color {number} the color to use to tint the sprite with
- * @return {HTMLCanvasElement} The tinted canvas
+ * @namespace PIXI.CanvasTinter
  */
-CanvasTinter.getTintedTexture = function (sprite, color)
-{
-    var texture = sprite.texture;
-
-    color = CanvasTinter.roundColor(color);
-
-    var stringColor = '#' + ('00000' + ( color | 0).toString(16)).substr(-6);
-
-    texture.tintCache = texture.tintCache || {};
-
-    if (texture.tintCache[stringColor])
+var CanvasTinter = module.exports = {
```

Comment:
The end result of the export is the same as before, but now that it is formatted in this way it is cleaner and jsdoc is parsing it correctly.